openslides_backend/presenter/check_database_all.py

```diff
@@ -64,9 +64,6 @@ def get_everything(self) -> Dict[str, Any]:
 def remove_meta_fields(res: Dict[int, Any]) -> Dict[str, Any]:
     dict_without_meta_fields = {}
     for key in res:
-        new_entry = {}
-        for fieldname in res[key]:
-            if not is_reserved_field(fieldname):
-                new_entry[fieldname] = res[key][fieldname]
-        dict_without_meta_fields[str(key)] = new_entry
+        strip_reserved_fields(res[key])
+        dict_without_meta_fields[str(key)] = res[key]
```

Comment:
Why do you copy into a new dict? Since this is a presenter, there should be no mutations on the dict and you can just change it in-place.

Reply:
I copy it, because of the int to str key transformation.